src/EnergyPlus/MixedAir.cc

```diff
@@ -3674,6 +3674,25 @@ namespace MixedAir {
         }
         this->MechVentOAMassFlowRequest = MechVentOAMassFlow;
         //****** use greater of Mechanical Ventilation Outside Air fraction and OutAirMinFrac
+        if (OutAirMinFrac > MechVentOutsideAirMinFrac) {
```

Comment:
@lgu1234 Sorry - I didn't look closely at the err diffs before. The new warning is tripping even for files that don't have a Controller:MechanicalVentilation object. 
This should be `if ((MechVentOutsideAirMinFrac > 0.0) && (OutAirMinFrac > MechVentOutsideAirMinFrac))`. That will also exclude times when the Controller:MechanicalVentilation object is scheduled off.

Reply:
@mjwitte Thanks for a good catch. I am going to make the change tomorrow.